package.json

```diff
@@ -45,6 +45,7 @@
     "puppeteer": "^1.19.0",
     "require-relative": "^0.8.7",
     "rollup": "^1.17.0",
+    "rollup-dependency-tree": "https://github.com/benmccann/rollup-dependency-tree.git",
```

Comment:
Yeah, I'll go ahead and publish if we're okay with it. Should I check with @Conduitry too or would your review be enough?

I'm not sure about adding `plugin` to the name because it's not actually a Rollup plugin in the sense that it's added to the `plugins` in `rollup.config.js`. It's really just a library

Reply:
published to npm and updated!